taxonomies/ingredients.txt

```diff
@@ -82853,6 +82866,12 @@ nl:gepasteuriseerd vloeibaar heel ei
 sv:helt pastöriserat ägg i flytande form
 # ingredient/fr:œuf-entier-liquide-pasteurisé has 143 products in 3 languages @2018-12-18
 
+#note: disabled for english translation due to conflict
+#en:pasteurised egg
+<en:whole egg
+ca:ou pasteuritzat
+es:huevo pasteurizado
+
```

Comment:
For this as well as for "ca:extracte de malta" above,
Open the Ingredients.pm file in lib/ProductOpener folder, and search for "# match before or after the ingredient, require a space".
Replace

`							or ($ingredients_lc eq 'cs')`
by

`							or ($ingredients_lc eq 'ca')
							or ($ingredients_lc eq 'cs')`

After having done this all ingredients in ingredients.txt can be completed with their processing in ingredients_processing.txt.

If you have "malta" for malt in ingredients.txt and "extracte de" in ingredients_processing.txt, "extracte de malta" should be recognized in the ingredient list of a product.
If you have "ou" for egg in ingredients.txt and "pasteuritzat" in ingredients_processing.txt, "ou pasteuritzat" should be recognized in the ingredient list of a product.

Reply:
There are 2 advantages to do it: 1) if later you have "extracte de something else" or "something else pasteuritzat" and "something else" is already in the taxonomy, then it will be recognized and will not need to add those in the taxonomy :) 
2) in the api ingredients will not be parsed as being "extract of malt" but as being "malt" having processing "extract of"

See for example (in Croatian), this product:
https://hr.openfoodfacts.org/product/3831051008991/desert-jogurtova-smetana-s-poslo%C5%BEenim-jagodami-zelene-doline
First ingredient is "Pasterizirano fermentirano vrhnje" "Pasteurized fermented cream"
It is not parsed as "Pasteurized fermented cream" but as "cream" having processing: "Pasteurized, fermented"
https://es.openfoodfacts.org/api/v2/product/3831051008991

Reply:
gotcha, thanks for the details!
I have removed `pasteurised egg` definition.